neo4j/src/main/java/org/hibernate/ogm/datastore/neo4j/impl/Neo4jIndexSpec.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Hibernate OGM, Domain model persistence for NoSQL datastores
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+
+package org.hibernate.ogm.datastore.neo4j.impl;
+
+import java.util.List;
+import java.util.Objects;
+
+import org.hibernate.ogm.datastore.neo4j.query.parsing.cypherdsl.impl.CypherDSL;
+
+import org.neo4j.graphdb.Label;
+
+/**
+ * @author The Viet Nguyen
+ */
+public class Neo4jIndexSpec {
```

Comment:
could you add implements Serializable?

Reply:
sure, but may I ask why should we do that? We don't move this anywhere outside the JVM yet. And, I don't see `MongoDBIndexSpec` implements `Serialzable` either

Reply:
It's an optional, sometimes it is better to mark a class Serializable, it a way to describe the nature of the class itself.

Reply:
I get it. Will add in the next push

Reply:
Now we are more focus on what is Serializable or not, more than in the past, for cloud compliance :P